### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by t-bello7**
- Setup rails project with webpack and postgres database
- Create table for storing greetings
- Implement api for randomly generating greeting
- Implement store, action and dipatcher for fetching api data

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,35 +166,16 @@ Feel free to check the [issues page](../../issues/).
...
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+I would like to thank the maintainers of ruby on rails, react and redux-toolkit for providing amazing open source programs
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (HAMMAS-SALEEM) [inline README.md:181]: image

- [x] Your link to your MIT.md file is broken :fearful: Kindly fix this link to make your readme descriptive :+1:

Comment (HAMMAS-SALEEM) [inline README.md:181]: image

- [x] Your license file is looking like a template. Kindly add your **name**, **current year** and **app-type** in the license file to make your project licensed :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @t-bello7 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @t-bello7,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**